plugins/lxc/lxc_ram

```diff
@@ -62,13 +70,24 @@ if [ "$1" = "config" ]; then
  echo 'graph_vlabel byte'
  echo 'graph_category lxc'
 
+ if [ "$areastack" = "true" ]; then
```

Comment:
Splitting the two conditional branches for areastack (line 73 and line 87) makes it quite hard to understand the logic of the code.
Maybe you want to change the code in a way to always output the mem_usage first and then follow with the other datasources. Each datasource will need a separate guest-loop in this case, but it will be easy to understand and maintain.